llvm/docs/ReleaseNotes.md

```diff
@@ -47,6 +47,12 @@ for adding a new subsection. -->
   same semantics. The normalizer makes it easier to spot semantic differences
   when diffing two modules which have undergone different passes.
 
+* The SPIR-V backend is now an official LLVM target, enabling support for 
+  SPIR-V across a range of APIs and use cases, including OpenCL, Vulkan, SYCL,
+  GLSL, and HLSL. This backend serves as a unified solution for diverse compute
+  and graphics workloads, offering a robust alternative to the Khronos SPIR-V
```

Comment:
How robust is the Vulkan support as of today? I understand that for opencl you track the pass rate on the CTS (https://discourse.llvm.org/t/rfc-promoting-spir-v-to-an-official-target/83614), but I wonder if we can quantify how robust the support if for other subtargets and whether we should add a disclaimer here.

Reply:
That is a very good point -- thanks for catching this! While both OpenCL and SYCL are conformant or nearly fully conformant. I don't think that is the case for Vulkan yet, there is a lot of work ahead. Folks at Google and Microsoft have more details on the current status and what the plans are.

I will rephrase the release note to better reflect this.